Project_Proposal.md

```diff
@@ -0,0 +1,30 @@
+### Name of the game: 
+# Brain on blue
+
+### Short description of gameplay: 
+Side scrolling  adventure game of a brain named BOB.
+Bob is searching for a head to inhabit but must travel
+overcome various obstacles to eventually find the head.
+
+### Win condition:
+BOB is able to navigate past all obsolesces and find a head
+
+### Lose condition:
+Obstacles cause enough damage that BOB dies
+
+### How the player interacts with the game:
+the interface will be via the keyboard using the "up", "Down",
+"left", and right arrow keys.
+
+### List of game features:
+* side scrolling adventure
```

Comment:
By this, do you mean the game is always automatically scrolling?

If you do want to do this, there will be a few things extra things to worry about during programming. Specifically, we need to figure out how BOB will interact with the edge of the screen.
 * Does it push him along?
 * If he's trapped between the edge of the screen and a wall, is he crushed?

It'll also involve actually programming the invisible wall and making sure the game camera moves at the same speed (probably following some invisible FlxObject at screen center).

Reply:
we spoke about having the obstacles have their own velocity across the screen, I will be using that in the development